Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.yml

```diff
@@ -0,0 +1,74 @@
+category: Endpoint
+defaultEnabled: true
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: Cortex Core - Platform
+  version: -1
+configuration:
+- additionalinfo: The timeout of the HTTP requests sent to Cortex API (in seconds).
+  defaultvalue: '120'
+  display: HTTP Timeout
+  name: timeout
+  type: 0
+  required: false
+  section: Connect
+description: This integration uses the Cortex API for accessing essential services and capabilities of the Cortex platform.
```

Comment:
Recommend:
This integration uses the Cortex API to access all the core services and capabilities of the Cortex platform.